src/ui/ResultList/ResultList.ts

```diff
@@ -225,11 +228,16 @@ export class ResultList extends Component {
      *
      * If you set the [`autoSelectFieldsToInclude`]{@link ResultList.options.autoSelectFieldsToInclude} option to
      * `true`, the Coveo Search API returns the fields you specify for this option (if those fields are available) in
-     * addition to the fields which the `ResultList` automatically requests.
+     * addition to the fields which the `ResultList` automatically requests. Note that the `autoSelectFieldsToInclude`
+     * option doesn't automatically request fields for folding templates; in such a case, you must manually specify 
+     * fields using this option to avoid empty results.
```

Comment:
Hmm, isn't this a bug in the JSUI that we should fix? It feels like `autoSelectFieldsToInclude` should work for folding templates too? Or did you discuss with someone from the team and decided to document instead? Was there a tech limitation that was difficult to overcome?

Reply:
@samisayegh we discussed it with @ThibodeauJF in the aforementioned PR https://github.com/coveo/doc_jekyll-public-site/pull/6364 (there's also a [JIRA ticket](https://coveord.atlassian.net/browse/DOC-9232)).
According to JF, this is not a bug but a feature. :)